public/client.js

```diff
@@ -514,22 +514,26 @@ function formatSizeUnits(size){
     return giga.toFixed(1) + " GB";
 }
 //makes all unrecognized files downloadable
-function unknown_file_dowloadable(attached){
+function unknown_file_downloadable(attached){
     let thegotten = getFileSize(attached)
-    return `<a href='${attached}'>${thegotten[1].split('"')[1]+" ("+thegotten[0]+")"}</a>`
+    return `<div class="downloadable_file_div"><a class="downloadable_file" target='_blank' href='${attached}'>${thegotten[1].split('"')[1]+" ("+thegotten[0]+")"}<img class="embed-dl-img" src='./assets/img/download_icon.svg' width=30></a>`
```

Comment:
```suggestion
    return `<div class="downloadable_file_div"><a class="downloadable_file" target='_blank' href='${attached}'>${thegotten[1].split('"')[1]+" ("+thegotten[0]+")"}<i class="fa-solid fa-download"></i></a>`
```
I forgot to do it here too :<